File: Documentation Code Snippets/ScanbotSDK Examples/MedicalCertificateScanner/DetectionOnImage/Objc/MCDetectionOnImage.m
//
//  MCDetectionOnImage.m
//  ScanbotSDK Examples
//
//  Created by Rana Sohaib on 27.03.23.
//

#import "MCDetectionOnImage.h"
@import ScanbotSDK;

@implementation MCDetectionOnImage

- (void) detectMedicalCertificateOnImage {
    
    // Image containing Medical Certificate.
    UIImage *image = [UIImage imageNamed:@"medicalCertificateImage"];
    
    // Creates an instance of `SBSDKMedicalCertificateRecognizer`.
    SBSDKMedicalCertificateRecognizer *detector = [[SBSDKMedicalCertificateRecognizer alloc] init];
    
    // Returns the result after running detector on the image.
    SBSDKMedicalCertificateRecognizerResult *result = [detector recognizeFromImage:image detectDocument:TRUE];
}

@end

File: Documentation Code Snippets/ScanbotSDK Examples/MedicalCertificateScanner/DetectionOnImage/Swift/MCDetectionOnImage.swift
//
//  MCDetectionOnImage.swift
//  ScanbotSDK Examples
//
//  Created by Rana Sohaib on 27.03.23.
//

import Foundation
import ScanbotSDK
    
func detectMedicalCertificateOnImage() {
    
    // Image containing Medical Certificate.
    guard let image = UIImage(named: "medicalCertificateImage") else { return }
    
    // Creates an instance of `SBSDKMedicalCertificateRecognizer`.
    let detector = SBSDKMedicalCertificateRecognizer()
    
    // Returns the result after running detector on the image.
    let result = detector.recognize(from: image, detectDocument: true)
}
